openhtf/util/units/__init__.py

```diff
@@ -0,0 +1,1170 @@
+# coding: iso-8859-1
```

Comment:
ah, I missed that the .xls file wasn't also getting checked in.  hm, it's a little riskier checking in the generated output instead of the xls because if the xls goes away, there's no way to get it back (whereas the other direction is easy).  i don't think lint is really a problem, just add disable comments where needed ;)

It's up to you which you prefer, i have a slight preference for checking in the xls and generating at build-time, but as long as both aren't checked in side by side, i'm okay with it.